src/SFA.DAS.AssessorService.Data/SettingRepository.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Dapper;
+using Microsoft.EntityFrameworkCore;
+using SFA.DAS.AssessorService.Application.Interfaces;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AssessorService.Data
+{
+    public class SettingRepository : Repository, ISettingRepository
+    {
+        public SettingRepository(IUnitOfWork unitOfWork)
+            : base(unitOfWork)
+        {
+        }
+
+        public async Task<string> GetSetting(string name)
+        {
+            const string sql = "SELECT [Value] FROM [Settings] WHERE [Name] = @Name";
+            var value = await _unitOfWork.Connection.QueryFirstOrDefaultAsync<string>(
+                sql,
+                param: new
+                {
+                    Name = name
+                },
+                transaction: _unitOfWork.Transaction);
+
+            return value;
+        }
+
+        public async Task SetSetting(string name, string value)
+        {
+            const string sql = "SELECT [Name] FROM [Settings] WHERE [Name] = @Name";
```

Comment:
It would be easier but about 95% of all the code looks similar to this currently - so I think it's best left looking as consistent as possible - I am adding stored procedure where they are overwhelming necessary though